src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Scripts/services/featureManagerWrapper.js

```diff
@@ -0,0 +1,33 @@
+var moduleName = "virtoCommerce.DemoSolutionFeaturesModule";
+
+angular.module(moduleName)
+    .factory('virtoCommerce.demoFeatures.featureManagerSubscriber', ['$rootScope', 'virtoCommerce.demoFeatures.featureManager', function ($rootScope, featureManager) {
+        var result = {};
+
+        result.callbackCollection = [];
+        result.subscribeToLoginAction = (featureName, callback) => {
+            if (!result.callbackCollection[featureName]) {
+                result.callbackCollection[featureName] = [];
+            }
+            if (typeof callback === 'function') {
+                result.callbackCollection[featureName].push(callback);
+            }
+        };
+
+        function initialize() {
+            $rootScope.$on('loginStatusChanged',
+                () => {
+                    for (const [featureName, callbacks] of Object.entries(result.callbackCollection)) {
+                        featureManager.isFeatureEnabled(featureName).then(() => {
```

Comment:
May we add comment how it works? I understand case what in case we are not authenticated feature will be disabled by feature manager, but I think it's not obvious

Reply:
done